coastlines/combined.py

```diff
@@ -291,7 +292,8 @@ def terrain_shadow(
     threshold: float = 0.25,
     radius: int = 1,
 ):
-    item = items_by_time[ds.time.values.astype(str).split(".")[0]]
+    item = items_by_time[ds.time.values.astype(str)[0].split(".")[0]]
```

Comment:
Possibly better to do this more explicitly, so:

```suggestion
    ds = ds.squeeze()
    item = items_by_time[ds.time.values.astype(str).split(".")[0]]
```

Reply:
Completed